packages/Webkul/Admin/src/Resources/views/sales/transactions/index.blade.php

```diff
@@ -204,10 +204,16 @@ class="text-gray-600 dark:text-gray-300"
                                     </p>
 
 
-                                    <p
-                                        class="text-gray-600 dark:text-gray-300"
-                                        v-text="data.status"
-                                    >
+                                    <p v-if="data.status === '@lang('admin::app.sales.transactions.index.datagrid.paid')'" class="label-active">
+                                        @lang('admin::app.sales.transactions.index.datagrid.paid')
+                                    </p>
+
+                                    <p v-else-if="data.status === '@lang('admin::app.sales.transactions.index.datagrid.pending')'" class="label-pending">
+                                        @lang('admin::app.sales.transactions.index.datagrid.pending')
+                                    </p>
+
+                                    <p v-else-if="data.status === '@lang('admin::app.sales.transactions.index.datagrid.completed')'" class="label-completed">
+                                        @lang('admin::app.sales.transactions.index.datagrid.completed')
```

Comment:
When a tag contains two or more attributes, we should separate each attribute onto a new line for better readability.

You can refer to the example code snippet below:

```
<p
    v-if="data.invoice_id"
    class="text-gray-600 dark:text-gray-300"
    v-text="data.invoice_id"
>
</p>
```